spec/helper/environment.js

```diff
@@ -29,6 +29,28 @@ EnvironmentTest.prototype.testCrypto = function () {
   });
 };
 
+EnvironmentTest.prototype.testRequire = function (tf) {
```

Comment:
Instead of a boolean, why don't we pass in the URL?
Then have the continuation return "success", "failure", and other various more descriptive results.